tonic/src/service/router.rs

```diff
@@ -47,6 +47,14 @@ impl RoutesBuilder {
         self.routes.unwrap_or_default()
     }
 }
+
+impl Default for Routes {
+    fn default() -> Self {
+        let router = axum::Router::new().fallback(unimplemented);
+        Self { router }
```

Comment:
Suggest inlining the `router` binding directly.

Reply:
Updated.